DependencyInjection/PropelExtension.php

```diff
@@ -129,6 +129,19 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
             $c['datasources']['default'] = $connectionName;
         }
 
+        // copy master connection setting to slave connections
+        foreach ($c['datasources'] as $datasource => $masterConnection) {
+            foreach (['options', 'attributes', 'settings'] as $attribute) {
+                if (!isset($c['datasources'][$datasource]['slaves'])) {
```

Comment:
Either `$datasource` should be `$attribute`, or this if statement can be moved one level up.